conans/test/functional/graph_lock/graph_lock_test.py

```diff
@@ -231,16 +233,21 @@ def package_info(self):
         self._check_lock("PkgB/0.1@")
 
         # If we create a new PkgA revision, for example adding info
-        client.save({"conanfile.py": str(TestConanFile("PkgA", "0.1", info=True))})
+        client.save({"conanfile.py":
+                         str(GenConanfile().with_name("PkgA").with_version("0.1")
+                             .with_package_info(
+                             cpp_info={"libs": ["mylibPkgA0.1lib"]},
+                             env_info={"MYENV": ["myenvPkgA0.1env"]}))})
+
         client.run("create . PkgA/0.1@user/channel")
         client.save({"conanfile.py": str(consumer)})
 
     def _check_lock(self, ref_b, rev_b=""):
         lock_file = load(os.path.join(self.client.current_folder, LOCKFILE))
         lock_file_json = json.loads(lock_file)
         self.assertEqual(2, len(lock_file_json["graph_lock"]["nodes"]))
-        self.assertIn("PkgA/0.1@user/channel#b55538d56afb03f068a054f11310ce5a:"
-                      "5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9#6d56ca1040e37a13b75bc286f3e1a5ad",
+        self.assertIn("PkgA/0.1@user/channel#fa090239f8ba41ad559f8e934494ee2a:"
```

Comment:
revision shouldn't change, should it?

Reply:
Revision changes if the content is different (this `conanfile.py` generator may add blank lines, change the order of things,...)